pysat/utils/files.py

```diff
@@ -203,26 +227,42 @@ def parse_fixed_width_filenames(files, format_str):
 def parse_delimited_filenames(files, format_str, delimiter):
     """Parse list of files, extracting data identified by format_str.
 
+    Will parse file using `delimiter` though the function does not require
+    every delimited item to be a variable, and more than one variable
```

Comment:
Every delimited item or every parsed item?

Reply:
I think delimited, it will work on filenames that use delimiters all over the place, and not related to variables. Like, 'example-that-delimits-everything-then-quits-2009-01-03.instrument.a.cdf'

If parsed makes more sense then I'll switch.

Reply:
All the previous tests run via the fixed_width parser are now also run using the delimited parser, even though none of those filenames were generated with delimiting in mind.